### ISSUE TITLE
<meteor-include src="loginButtons"></meteor-include> does not work

### ISSUE THREAD

**Issue by nigel-dewar** [Urigo/angular-meteor#707]
Hi There

Not sure why, but since meteor 1.2 meteor-include directive displays nothing.

**Comment by Urigo** [comment]
Yes, it has been deprecated in favor of `blaze-template`:  http://angular-meteor.com/api/blaze-template
You only need to do:

    meteor add urigo:angular-blaze-template

    <blaze-template
        name="loginButtons">
    </blaze-template>

**Comment by nigel-dewar** [comment]
Thanks Uri,

 

My bad. You do have thus in your instructions here

 

http://angular-meteor.com/tutorials/angular1/user-accounts-authentication-and-permissions

 

but just so you know in code snippet, 8.1 the old method is still included as well.

 



 

Cheers bro. love your work.

**Comment by Urigo** [comment]
@nigel-dewar thanks!  I've just fixed the tutorial and pushed a new update.

**Comment by torayeff** [comment]
not sure why but when I include <blaze-template name="loginButtons"></blaze-template> and press it the dropdown does not show the login menu.  Instead {{> loginButtons}} just works fine. Maybe there is issue in meteor 1.2.1?

**Comment by Urigo** [comment]
@torayeff it might be the new `urigo:angular-blaze-template` package version.
Try to downgrade to `0.0.1` for now and let me know if that helped

**Comment by torayeff** [comment]
@Urigo yes the problem was with the new version. I downgraded and now it is fine. But after updating angular package (1.0.4 => 1.0.5) I started receiving this error in browser console: 
```update failed: Access denied. In a restricted collection you can only update documents, not replace them. Use a Mongo update operator, such as '$set'.```

I am following angular-meteor tutorial, step 8.

**Comment by Urigo** [comment]
@torayeff can you share your code in a repo?

**Comment by torayeff** [comment]
Now everything is working for me with angular package 1.0.6 and urigo:angular-blaze-template package 0.1.1. Thanks.

**Comment by torayeff** [comment]
Hi @Urigo . For this:
```
$stateProvider
  .state('parties', {
    url: '/parties',
    templateUrl: 'parties-list.html',
    controller: 'PartiesListCtrl'
  })
  .state('partyDetails', {
    url: '/parties/:partyId',
    templateUrl: 'party-details.html',
    controller: 'PartyDetailsCtrl'
  });
```

If I provide templateUrl: 'client/parties/views/parties-list.html', I receive this error:
```WARNING: Tried to load angular more than once.```

My repo is here https://github.com/torayeff/angular-meteor-bootstrap

**Comment by Urigo** [comment]
Looks like a typo in path or a caching problem.
The caching problem should be solved in the new [1.3 version](http://info.meteor.com/blog/angular-meteor-1.3)
